docker-entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
```

Comment:
It might be a good idea to add `set -e` so this exits upon any failure as that could lead to some unexpected behavior.